platforms/tktrex/backend/bin/app.ts

```diff
@@ -148,29 +148,30 @@ export const makeApp = async (
     '/api/v2/guardoni/list/:directiveType/:key?',
     async (req, res) => await iowrapper('getAllExperiments', req, res)
   );
-  app.get('/api/v3/directives/public', async (req, res) =>
```

Comment:
This changes needs to be reflected in the `@tktrex/shared/endpoints` - and one day we would implement the endpoint from their definitions